### PR TITLE
Remove old-style cme_registration

### PR DESCRIPTION
**Pull request by stvstnfrd**
This is incompatible with logistration and has been deprecated in favor
of the `register_cme` djangoapp, which we've been running in production
for nearly a year now.

### CODE REVIEW

**Comment by caesar2164** [comment]
nice net code change! 👍

**Comment by stvstnfrd** [comment]
@dcadams @caesar2164 Sorry to pull you back, but just prior to merging I noticed that there were still a few more references. I have added an additional commit [1], if you wouldn't mind a quick double-check. I'll squash before merging.

[1] 1e98dc59578611b853c1b744a6961f878b0c132b

File: lms/djangoapps/instructor/management/commands/dump_userinfo.py [hunk 1]
```diff
@@ -23,70 +22,37 @@
...
+    ('user__extrainfo__state', 'State'),
+    ('user__extrainfo__postal_code', 'Postal Code'),
+    ('user__extrainfo__county_province', 'County/Province'),
+    ('user__extrainfo__country', 'Country'),
+    ('user__extrainfo__phone_number_untracked', 'Phone Number'),
+    ('user__profile__gender', 'Gender'),
+    ('user__extrainfo__marketing_opt_in_untracked', 'Marketing Opt-In'),
+    ('user__id', ''),
 ]
-if 'openedx.stanford.djangoapps.register_cme' in settings.INSTALLED_APPS:
```
Comment (stvstnfrd) [inline lms/djangoapps/instructor/management/commands/dump_userinfo.py]: Just make this the default case now.

File: lms/djangoapps/instructor/management/commands/dump_userinfo.py [hunk 2]
```diff
@@ -305,15 +269,6 @@ def handle(self, *args, **options):
...
     def query_database_for(self, course_id):
-        cme_profiles = CourseEnrollment.objects.select_related('user__profile__cmeuserprofile').filter(course_id=course_id).values(
-            *[field for field, label in PROFILE_FIELDS if 'untracked' not in field]
-        ).order_by('user__username')
-        unpaid_registrations = CourseEnrollment.objects.filter(course_id=course_id)
-        registrations = PaidCourseRegistration.objects.filter(status='purchased', course_id=course_id)
-        certificates = GeneratedCertificate.objects.filter(course_id=course_id)
-        return certificates, cme_profiles, registrations, unpaid_registrations
-
-    def query_database_for_new_app(self, course_id):
```
Comment (stvstnfrd) [inline lms/djangoapps/instructor/management/commands/dump_userinfo.py]: We no longer need the `_new_app` suffix.

File: lms/djangoapps/instructor/management/commands/dump_userinfo.py [hunk 3]
```diff
@@ -215,8 +181,6 @@ def handle(self, *args, **options):
         sys.stdout.write("Fetching enrolled students for {course}...".format(course=course_id))
 
         certificates, profiles, registrations, unpaid_registrations = self.query_database_for(course_id)
-        if 'openedx.stanford.djangoapps.register_cme' in settings.INSTALLED_APPS:
```
Comment (stvstnfrd) [inline lms/djangoapps/instructor/management/commands/dump_userinfo.py]: Just make this the default case now.

File: lms/djangoapps/instructor/management/commands/dump_userinfo.py [hunk 4]
```diff
@@ -22,70 +22,37 @@
...
+    ('user__extrainfo__address_1', 'Address 1'),
+    ('user__extrainfo__address_2', 'Address 2'),
+    ('user__extrainfo__city', 'City'),
+    ('user__extrainfo__state', 'State'),
+    ('user__extrainfo__postal_code', 'Postal Code'),
+    ('user__extrainfo__county_province', 'County/Province'),
+    ('user__extrainfo__country', 'Country'),
+    ('user__extrainfo__phone_number_untracked', 'Phone Number'),
+    ('user__profile__gender', 'Gender'),
+    ('user__extrainfo__marketing_opt_in_untracked', 'Marketing Opt-In'),
```
Comment (caesar2164) [inline lms/djangoapps/instructor/management/commands/dump_userinfo.py]: Why are we renaming these? This still remains a fork with this commit applied, so wouldn't it be better to keep the values that state it's for CME?

Comment (stvstnfrd) [reply lms/djangoapps/instructor/management/commands/dump_userinfo.py]: @caesar2164 No, that's not what's happening.

We used to default `PROFILE_FIELDS` to the columns for the old app,
then selectively override if the new app were installed.

Now we just default to the new app values.

Comment (caesar2164) [reply lms/djangoapps/instructor/management/commands/dump_userinfo.py]: ah ok.

**Pull request merged**